latte/latte.Config.json

```diff
@@ -0,0 +1,39 @@
+{
+    "LatteDefault": 
+    {
+        "Iterations"     : 5,
+        "Protocol"       : ["tcp", "udp"],
+        "StartPort"      : 50000,
+        "Time"           : 30,
+        "PingIterations" : 10000,
+        "SendMethod"     : [ "b" ],
+        "Default"        : null,
+        "Optimized"      : "-group 0 -rio -riopoll 100000000000",
+        "Options"        : ""
+    },
+
+    "LatteAzure": 
+    {
+        "Iterations"     : 15,
+        "Protocol"       : ["tcp", "udp"],
+        "StartPort"      : 50000,
+        "Time"           : 30,
+        "PingIterations" : 10000,
+        "SendMethod"     : [ "b" ],
+        "Default"        : null,
+        "Optimized"      : "-group 0 -rio -riopoll 100000000000",
+        "Options"        : ""
+    },
+
+    "LatteDetail": {
+        "Iterations"     : 5,
+        "Protocol"       : ["tcp", "udp"],
+        "StartPort"      : 50000,
+        "Time"           : 10,
+        "PingIterations" : 20000,

```

Comment:
might want to consult with Omar about why this is twice as much as the pingiteration values in the other two config files?

Reply:
I don't have context on PingIterations.  It wasn't in my sample file.

Reply:
It is the bounded iterations tests (-i option). Sorry this is a typo from linux, should be 10000.